src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,15 @@
+package calculator
+
+class StringAddCalculator {
+    fun calculate(expression: String?): Long {
+        if (expression.isNullOrEmpty()) {
+            return DEFAULT_VALUE
+        }
+
+        return StringParser().parsingNumbers(expression).sum()
```

Comment:
메서드를 연속해서 호출할 경우에는 행 구분을 해주세요!

https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

```suggestion
        return StringParser().parsingNumbers(expression)
            .sum()
```